src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -23,6 +23,10 @@
  */
 abstract class BaseNode implements NodeInterface
 {
+    const DEFAULT_PATH_SEPARATOR = '.';
+
+    private $pathSeparator;
```

Comment:
since all other props are protected, should we make this one protected also, and use it directly in child nodes?